torchgen/selective_build/selector.py

```diff
@@ -233,6 +233,11 @@ def is_kernel_dtype_selected(self, kernel_tag: str, dtype: str) -> bool:
     def is_et_kernel_selected(self, op_name: str, kernel_key: str) -> bool:
         if op_name not in self.et_kernel_metadata:
             return False
+
+        # Always select default key for now
+        if kernel_key == "default":
+            return True
+
```

Comment:
Hmm are you sure this is the right logic? I think all of the model kernel key should have a kernel, but we may not need to register a "default" kernel from the codegen perspective, right?

I think we need extra tooling to make sure all kernel keys from a model should have a kernel, and that should be a part of kernel registration UX work.

Reply:
Right!  I will fix it later.  Seems that we need additional information to induce it.